### PR TITLE
3D Bounding box stuff

### PR DESCRIPTION
**Pull request by rohitvarkey**
Changes made:
- Added definitions for 3D Bounding Boxes 
- Defined d as a Length constant like w and h.
- depth function to access the depth of the box.
- Added a resolve method to resolve measures w.r.t to depth

### CODE REVIEW

**Comment by shashi** [comment]
Looks like this needs a rebase. Could you look into it?

**Comment by rohitvarkey** [comment]
I've rebased it with `master`

File: src/boundingbox.jl [hunk 1]
```diff
@@ -23,5 +30,19 @@ typealias AbsoluteBox{N}   BoundingBox{NTuple{N, Length{:mm, Float64}},
...
+                                    AbsoluteLength,
+                                    AbsoluteLength
+                                },
+                                Tuple{
+                                    AbsoluteLength,
+                                    AbsoluteLength,
+                                    AbsoluteLength
+                                }
+                            }
+
```
Comment (shashi) [inline src/boundingbox.jl:45]: Fancy indentation ;)

Comment (rohitvarkey) [inline src/boundingbox.jl:45]: LOL.

**Pull request merged**